aip/0216.md

```diff
@@ -60,13 +60,57 @@ The field referencing the `State` enum in a resource **should** behave and be
 documented as "Output only", in accordance with [AIP-203](./0203.md).
 
 APIs **should not** allow a `State` enum to be directly updated through an
-"update" method, and **should** instead use a custom method.
+"update" method, and **should** instead use a custom state transition method.
 
 This is because update methods are generally not expected to have side effects,
 and also because updating state directly implies that it is possible to set the
 state to any available value, whereas states generally reflect a resource's
 progression through a lifecycle.
 
+### State transition methods
+
+State transition methods should look like this:
+
+```proto
+// Publishes a book.
+// The `state` of the book after publishing will be PUBLISHED.
+rpc PublishBook(PublishBookRequest) returns (Book) {
+  option (google.api.http) = {
+    post: "/v1/{name=publishers/*/books/*}:publish"
+    body: "*"
+  };
+}
+```
+
+- The name of the method **should** be a verb followed by the singular form of
+  the resource's message name.
+- The request message **must** match the RPC name, with a `-Request` suffix.
+- The response message **must** be the resource itself.
```

Comment:
What about LROs?

Reply:
Added

Reply:
The copy and paste here needs some love. It refers to create RPCs, for example.
Also, I suggest linking to AIP-151.

Reply:
Oops - fixed!